default.lcf

```diff
@@ -108,3 +108,4 @@ log4j.category.javax.jmdns=OFF
 
 #log4j.category.jmri.jmrit.display.layoutEditor=DEBUG
 #log4j.category.jmri.jmrit.display.layoutEditor.LayoutBlock=DEBUG
+log4j.category.jmri.jmrit.roster.PrintRosterEntry=DEBUG
```

Comment:
Can you please revert this change? I'm assuming this was committed in error (I make mistakes like this frequently myself as well).